float64 Linear
float64 Radius
